---a/net/minecraft/block/BlockEventData.java
+++b/net/minecraft/block/BlockEventData.java
@@ -2,58 +2,48 @@
 
 import net.minecraft.util.BlockPos;
 
-public class BlockEventData
-{
+public class BlockEventData {
+
     private BlockPos position;
     private Block blockType;
     private int eventID;
     private int eventParameter;
     private static final String __OBFID = "CL_00000131";
 
-    public BlockEventData(BlockPos pos, Block blockType, int eventId, int p_i45756_4_)
-    {
+    public BlockEventData(BlockPos pos, Block blockType, int eventId, int p_i45756_4_) {
         this.position = pos;
         this.eventID = eventId;
         this.eventParameter = p_i45756_4_;
         this.blockType = blockType;
     }
 
-    public BlockPos getPosition()
-    {
+    public BlockPos getPosition() {
         return this.position;
     }
 
-    public int getEventID()
-    {
+    public int getEventID() {
         return this.eventID;
     }
 
-    public int getEventParameter()
-    {
+    public int getEventParameter() {
         return this.eventParameter;
     }
 
-    public Block getBlock()
-    {
+    public Block getBlock() {
         return this.blockType;
     }
 
-    public boolean equals(Object p_equals_1_)
-    {
-        if (!(p_equals_1_ instanceof BlockEventData))
-        {
+    public boolean equals(Object p_equals_1_) {
+        if (!(p_equals_1_ instanceof BlockEventData)) {
             return false;
-        }
-        else
-        {
-            BlockEventData blockeventdata = (BlockEventData)p_equals_1_;
+        } else {
+            BlockEventData blockeventdata = (BlockEventData) p_equals_1_;
             return this.position.equals(blockeventdata.position) && this.eventID == blockeventdata.eventID && this.eventParameter == blockeventdata.eventParameter && this.blockType == blockeventdata.blockType;
         }
     }
 
-    public String toString()
-    {
+    public String toString() {
         return "TE(" + this.position + ")," + this.eventID + "," + this.eventParameter + "," + this.blockType;
     }
-}
+}
